docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -21,7 +21,9 @@ For Servlet support, Quarkus employs a customized Undertow version that operates
 When Undertow is present, RESTEasy functions as a Servlet filter.
 In its absence, RESTEasy operates directly on Vert.x without involving Servlets.
 
-== Serving Static Resources
+== Serving static resources
```

Comment:
Having a section about static resources definitely makes sense to have in here but this section starts rather abruptly and start referring to compression and other config which is not introduced before later.

Maybe move this down a bit ?

Reply:
This doc was already there, I just removed the capitalisation and added a link to the web doc

Reply:
yes you are right - just its starting to "grow" big. not a blocker for now.